docker/config/magpie/config.dev.yml

```diff
@@ -0,0 +1,58 @@
+## Combined Configuration
```

Comment:
Instead of the 2 files that modify the hostname, how about a single one that generates them in combination with `envsubst` called in the corresponding makefile target before starting compose?

Reply:
It looks like a lot of trouble for something that could require more differences in the future

Reply:
I keep the cowbird.example.ini as is (since it is also used by pycharm), but the docker-up makefile target creates a cowbird.docker.ini on the fly using the good hostname.